packages/forma-36-react-components/src/components/TextInput/TextInput.css

```diff
@@ -93,7 +93,6 @@
   & button {
     border-left: none;
     height: 100%;
-    background-color: var(--color-white);
```

Comment:
This seems like one of those changes that might have bigger implications than we think 🙂 How does this look with the different editors and apps?

Reply:
+1

Reply:
@denkristoffer the input with CopyButton does not change the look. It was changed to white when we did the v3 refresh. now I just remove this line, since this white background-color is now coming from `CopyButton`.